package.json

```diff
@@ -57,14 +52,6 @@
     "supertest": "^6.3.3",
     "wait-on": "^6.0.1"
   },
-  "standard": {
-    "plugins": [
-      "cypress"
-    ],
-    "env": {
-      "cypress/globals": true
-    }
-  },
   "jest": {
     "testPathIgnorePatterns": [
       "__tests__/spec/utils.js",
```

Comment:
@36degrees Thanks for all this 🙌

Got an outdated ignore pattern here now (one of the removed files)

Reply:
Good spot!